app/src/ui/diff/text-diff.tsx

```diff
@@ -401,6 +415,11 @@ export class TextDiff extends React.Component<ITextDiffProps, {}> {
   private onDocumentMouseUp = (ev: MouseEvent) => {
     ev.preventDefault()
 
+    // Discard right click events
```

Comment:
This comment read to me as  "ignore right click events". Might I suggest the following rewording?

```suggestion
    // We only care about the primary button here, secondary
    // button clicks are handled by `onContextMenu`
```